content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -31,32 +23,33 @@ Each `node_size` has a corresponding requirement for available RAM. For example,
 
 To tune your server, it may be useful to start with `tiny` and increase the size to `small`, `medium`, and so on as you refine the requirements for your use case.
 
-***TODO: Small thing, but I switched the order of the node size and RAM columns because the node size column contains the value that you are setting and should be the starting point of focus.***
-
-| `node_size` value | Available RAM for `rippled` | Notes                      |
+| `node_size` value | RAM available for `rippled` | Notes                      |
```

Comment:
I think the column headers are still swapped?

Reply:
Facepalm. Fixing now!